scripts/dockerbot/dockerbot.py

```diff
@@ -420,14 +402,11 @@ def enc_key(self):
     def create_pytest_envvars(self):
         self.env_details = {'BROWSER': self.args['browser'],
                             'CFME_CRED_REPO': self.args['cfme_cred_repo'],
-                            'CFME_CRED_REPO_DIR': self.args['cfme_cred_repo_dir'],
```

Comment:
you can still test against own repos, it will checked out as repo/branch under test